data/src/main/java/com/bottlerocketstudios/brarchitecture/data/model/CommitDto.kt

```diff
@@ -15,5 +15,6 @@ data class CommitDto(
     @Json(name = "type") val type: String?,
     @Json(name = "hash") val hash: String?,
     @Json(name = "author") val author: AuthorDto?,
-    @Json(name = "repository") val commitRepository: CommitRepositoryDto?
+    @Json(name = "repository") val commitRepository: CommitRepositoryDto?,
+    var branchName: String?
```

Comment:
@LuisRocketRangel We should avoid adding additional data to Dto classes, even moreso mutable data. Going to work on an alternate solution and push up a review commit later today